### ISSUE TITLE
Getting weird behavior on legacy css files

### ISSUE THREAD

**Issue by PepijnSenders** [jakubpawlowicz/clean-css#653]
This part of code will break with the following version of `"clean-css": "3.4.0"`

```
.footer_container .size1of2 {
    *width:49%;
    width:49%\9;
}
```

After doing some logging myself I found out it was being thrown in the `lib/properties/restore-from-optimizing.js` file on line `53`.

Adding some `console.log's` to that part of the code let me find out that whenever `\9` occured in the code. The property `position` of the `property` object, was one too many and that will leave `var current` on line `52` `undefined`. Resulting in the following error:

`TypeError: Cannot read property '0' of undefined`

**Comment by PepijnSenders** [comment]
This apparently was fixed in `3.4.1`.

**Comment by jakubpawlowicz** [comment]
Yup, sorry about that. Hopefully not so many people will run into it as 3.4.1 fix was posted just couple hours after 3.4.0.
